googleapis/google-cloud-node
# migrate code from googleapis/nodejs-data-fusion

- feat: initial stub of library
- feat: add templated files from docker image
- feat: add tests and samples to library (#1)
- chore: update package name for npm data-fusion (#3)
- chore: release 0.2.0 (#5)
- docs: point to reference docs (#6)
- chore(deps): update dependency sinon to v11 (#8)
- chore(deps): update dependency mocha to v9 (#7)
- docs: update reference api url (#10)
- build(node): update deps used during postprocessing (#1243) (#9)
- chore: fix the wrong post processor image (#14)
- chore(deps): update dependency @types/node to v16 (#13)
- chore(deps): update dependency sinon to v12 (#15)
- docs(samples): add example tags to generated samples (#16)
- build: add generated samples to .eslintignore (#17)
- docs(node): support "stable"/"preview" release level (#1312) (#20)
- chore: add api_shortname and library_type to repo metadata (#19)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#22)
- test(nodejs): remove 15 add 16 (#1322) (#24)
- chore: update v2.12.0 gapic-generator-typescript (#28)
- chore(deps): update dependency sinon to v13 (#30)
- docs(nodejs): version support policy edits (#1346) (#32)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#33)
- chore: update v2.14.2 gapic-generator-typescript (#37)
- feat: promote to stable (#38)
- chore(main): release 1.0.0 (#39)
- build(node): update client library version in samples metadata (#1356) (#45)
- chore(deps): update dependency sinon to v14 (#48)
- build!: update library to use Node 12 (#50)
- chore(main): release 2.0.0 (#51)
- chore(deps): update dependency jsdoc-fresh to v2 (#53)
- chore(deps): update dependency jsdoc-region-tag to v2 (#54)
- feat: support regapic LRO (#55)
- chore(main): release 2.1.0 (#56)
- chore(deps): update dependency linkinator to v4 (#60)
- fix: better support for fallback mode (#61)
- fix: change import long to require (#62)
- fix: do not import the whole google-gax from proto JS (#1553) (#65)
- fix: allow passing gax instance to client constructor (#66)
- chore(main): release 2.1.1 (#63)
- fix: preserve default values in x-goog-request-params header (#67)
- test: use fully qualified request type name in tests (#70)
- fix(deps): use google-gax v3.5.2 (#74)
- chore(deps): update dependency @types/node to v18 (#72)
- fix: regenerated protos JS and TS definitions (#77)
- chore(deps): update dependency jsdoc to v4 (#76)
- chore(main): release 2.1.2 (#68)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
